template-compose/Dangerfile

```
@@ -0,0 +1,49 @@
+lint_dir = "**/**/build/reports/lint/lint-result.xml"
+Dir[lint_dir].each do |file_name|
+  android_lint.skip_gradle_task = true
+  android_lint.report_file = file_name
+  android_lint.lint(inline_mode: true)
+end
+  
+# Show Danger test coverage report from Kover for templates
+# Report coverage of modified files, warn if total project coverage is under 80%
+# or if any modified file's coverage is under 95%
+
+kover_file_template = "app/build/reports/kover/report.xml"
```

RuboCop (1.82.1)

[warning] Useless assignment to variable - `kover_file_template`. Did you mean `kover_file_template_compose`?

(Lint/UselessAssignment)